### PR TITLE
Admin gives authority to publish auctions over $3500

### PR DESCRIPTION
**Pull request by jessieay**
* Contracting officer can create auctions over $3500
* Regular admins can only create auctions up to $3500
* Add `user` foreign key to auctions to enforce validations
* Closes https://github.com/18F/micropurchase/issues/411
* Closes https://github.com/18F/micropurchase/issues/392

### CODE REVIEW

File: app/controllers/admin/auctions_controller.rb [hunk 1]
```diff
@@ -37,37 +36,43 @@ def new
...
+      render_errors(@auction.errors.full_messages.to_sentence, :new)
     end
-  rescue ArgumentError => e
-    respond_error(e)
   end
 
   def update
     auction = Auction.find(params[:id])
-    UpdateAuction.new(auction, params).perform
+    UpdateAuction.new(auction, params, current_user).perform
```
Comment (harrisj) [inline app/controllers/admin/auctions_controller.rb]: I like this

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -31,3 +31,10 @@
     mock_github
   end
 end
+
+Shoulda::Matchers.configure do |config|
```
Comment (harrisj) [inline spec/rails_helper.rb]: Are we using shoulda anywhere in here yet or is this for the future?

**Comment by harrisj** [comment]
LGTM, merging

File: spec/rails_helper.rb (hunk 2 above)
Comment (jessieay) [inline spec/rails_helper.rb]: added a couple shoulda specs in `auction_spec` in this PR

File: app/controllers/admin/auctions_controller.rb (hunk 1 above)
Comment (jessieay) [inline app/controllers/admin/auctions_controller.rb]: 👍

**Pull request merged**